axlearn/audio/asr_encoder_test.py

```diff
@@ -28,6 +29,7 @@ class SpeechFeatureLayerTest(TestCase):
     """Tests SpeechFeatureLayer."""
 
     @parameterized.parameters([True, False])
+    @pytest.mark.fp64
```

Comment:
I wonder why you see these changes? Is this rebased to main?

Reply:
Now solved.